batch/batch/driver/main.py

```diff
@@ -1422,6 +1573,10 @@ async def on_startup(app):
     task_manager.ensure_future(periodically_call(15, monitor_system, app))
     task_manager.ensure_future(periodically_call(5, refresh_globals_from_db, app, db))
 
+    # 3 seconds was chosen for the initial compaction time to get through all existing records in a reasonable amount of time
+    task_manager.ensure_future(periodically_call(1, compact_agg_billing_project_users_table, db))
+    task_manager.ensure_future(periodically_call(1, compact_agg_billing_project_users_by_date_table, db))
```

Comment:
Given that the query to find a record takes 2-15s, maybe we should do this every 60s?